.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,77 @@
+name: Build and test
+
+on:
+    # Only on pushes to master
+    push:
+        branches:
+            - master
+        tags:
+    # Build pull requests
+    pull_request:
+
+jobs:
+    test:
+        strategy:
+            matrix:
+                py:
+                    - "2.7"
+                    - "3.6"
+                    - "3.7"
+                    - "3.8"
+                    - "3.9"
+                    - "pypy-2.7"
+                    - "pypy3"
+                os:
+                    - "ubuntu-latest"
+                architecture:
+                    - x64
+
+        name: "Python: ${{ matrix.py }}-${{ matrix.architecture }} on ${{ matrix.os }}"
+        runs-on: ${{ matrix.os }}
+        steps:
+            - uses: actions/checkout@v2
+            - name: Setup python
+              uses: actions/setup-python@v2
+              with:
+                  python-version: ${{ matrix.py }}
+                  architecture: ${{ matrix.architecture }}
+            - run: pip install tox
+            - name: Running tox
+              run: tox -e py
+    coverage:
+        runs-on: ubuntu-latest
+        name: Validate coverage
+        steps:
+            - uses: actions/checkout@v2
+            - name: Setup python 3.9
+              uses: actions/setup-python@v2
+              with:
+                  python-version: 3.9
+                  architecture: x64
+
+            - run: pip install tox
+            - run: tox -e py27,py39,coverage
+#    docs:
```

Comment:
I've created a separate issue to track this. Note that Sphinx 4.0 might break a docs build. If needed, temporarily pin to <4.0, and open an issue in the relevant project.

Reply:
https://github.com/Pylons/pyramid_layout/issues/39